### PR TITLE
Introduce Packit service and Testing Farm

### PR DESCRIPTION
**Pull request by mildas**
#### Description:
Build RPM on CentOS Stream 8 and 9 and on Fedora.
Run all tests from https://src.fedoraproject.org/tests/scap-security-guide on those rpm builds.

### CODE REVIEW

File: .packit.yaml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    - fedora-all-x86_64
+    - centos-stream-8-x86_64
+    - centos-stream-9-x86_64
+
+- job: tests
+  trigger: pull_request
+  metadata:
+    targets:
+    - fedora-latest-x86_64
+    - centos-stream-8
```
Comment (matusmarhefka) [inline .packit.yaml:23]: Let's unify the architectures, either do not specify them or use x86 everywhere.

File: packit.fmf [hunk 2]
```diff
@@ -0,0 +1,5 @@
+discover:
+  how: fmf
+  url: https://src.fedoraproject.org/tests/scap-security-guide.git
```
Comment (matusmarhefka) [inline packit.fmf:3]: What about adding `filter: 'tag: Packit'` so we would need to approve which tests are selected for gating by adding `Packit` tag to them?

File: .packit.yaml (hunk 1 above)
Comment (mildas) [reply .packit.yaml:23]: I have removed architecture from `fedora-latest`.

File: packit.fmf (hunk 2 above)
Comment (mildas) [reply packit.fmf:3]: I agree. Filter is there and I have also updated smoke-test with the new `Packit` tag

**Review by matusmarhefka — APPROVED** [review]

**Pull request merged**